pkg/scheduler/apis/config/v1alpha1/defaults.go

```diff
@@ -38,8 +38,8 @@ func addDefaultingFuncs(scheme *runtime.Scheme) error {
 	return RegisterDefaults(scheme)
 }
 
-// SetDefaults_KubeSchedulerConfiguration sets additional defaults
-func SetDefaults_KubeSchedulerConfiguration(obj *kubescedulerconfigv1alpha1.KubeSchedulerConfiguration) {
+// SetDefaultsKubeSchedulerConfiguration sets additional defaults
+func SetDefaultsKubeSchedulerConfiguration(obj *kubescedulerconfigv1alpha1.KubeSchedulerConfiguration) {
```

Comment:
Thanks @bagrat 
But please do not modify these `SetDefaults_XXXX` functions.
This naming format has a special meaning and is used for automatic code generation.

Ref: https://github.com/kubernetes/code-generator/blob/master/cmd/conversion-gen/generators/conversion.go#L80